### ISSUE TITLE
Invalid 'IN' expression created on converting AST to SQL 

### ISSUE THREAD

**Issue by corinv** [oguimbal/pgsql-ast-parser#35]
Currently having a trouble with `IN` expressions. When converting an AST with an `IN` expression the output contains an additional set of parentheses around the right hand side, which breaks the query.

Small reproduction:

```
const pap = require('pgsql-ast-parser');

const query = `SELECT * FROM "tbl" WHERE "id" IN (1,2,3,4,5)`;

const ast = pap.parseFirst(query);

const result = pap.toSql.statement(ast);

console.log(result);
```

Which outputs:

`SELECT *  FROM "tbl"   WHERE ("id" IN (((1), (2), (3), (4), (5))))`

Which if run, throws this error:

`operator does not exist: integer = record`

due to (from what I can gather) is the extra parentheses turning the right hand expression into a list of records.

Thanks for your work on this lib, has worked like a charm for us aside from this issue!

**Comment by corinv** [comment]
Dug around a little more and I think the cause is here:

https://github.com/oguimbal/pgsql-ast-parser/blob/2e34c15fe966af14f06f6bd74438b5be9a0bedc4/src/to-sql.ts#L339-L343

Locally tried adding this as a naive solution:

```
if (e.type === 'list') {
    m.super().expr(e);
    return;
}
```

And all the current tests still pass, however I'm unsure if there are cases where this could break precedence.

**Comment by oguimbal** [comment]
Thanks for the investigation.
Adding parenthesis everywhere was the lazy way to implement precedence. But it would seem you hit a limit of this hack !

I think that your fix would work.
I cannot think of any case where it would break precedence: There are parenthesis around a list expression... nothing beats that :)

The clean fix would be to implement actual precedence (and insert parenthesis only when needed), but I have other priorities right now :)


Do you want to create a pull request ? Or should I publish your quickfix myself ? (I dont mind either way... you creating a pull request will give you the credit, but requires a bit of effort from you !)

**Comment by corinv** [comment]
Thanks for the swift reply! Have just opened a PR containing the fix at #36
